stories/helperComponents/MockSettings.scss

```diff
@@ -1,57 +1,82 @@
-.numberPickerContainer {
-  display: flex;
-  margin: 50px;
-}
+$colorPickerWidth: 210px;
+$colorPickerMargin: 5px;
+$palettePickerMarginLeft: 96px;
+$palettePickerWidth: 96px + 12px + $palettePickerMarginLeft;
 
-.fontPickerContainer {
-  display: flex;
-  margin: 50px;
-}
+.root {
+  .styleItem {
+    padding: 50px;
 
-.colorPalettePicker {
-  display: flex;
-  margin: 50px;
-  position: relative;
-  width: fit-content;
-}
+    .title {
+      margin-bottom: 24px;
+    }
 
-.colorPickerContainer {
-  width: fit-content;
-  max-width: 520px;
-  overflow-x: auto;
+    .numberPickerContainer {
+      display: flex;
+    }
 
-  .pickerList {
-    width: max-content;
+    .fontPickerContainer {
+      display: flex;
+    }
 
-    > li {
-      width: 210px;
+    .colorPalettePicker {
+      position: relative;
+      min-height: 150px;
       display: inline-block;
-      margin: 0 5px;
-    }
-  }
-}
+      padding-right: $palettePickerWidth;
 
-.palettePickerContainer {
-  min-height: 150px;
-  overflow-y: auto;
-  position: absolute;
-  right: -150px;
-  top: 0;
-  bottom: 0;
-}
+      .colorPickerContainer {
+        max-width: ($colorPickerWidth + ($colorPickerMargin * 2)) * 3;
+
+        .pickerList {
```

Comment:
not critical but whenever i see i must comment :)
try to avoid nesting as much as possible, css should not reflect html hierarchy. with css modules nesting is almost never required

Reply:
I respectfully disagree 😄 
In any case, this is just the style for the storybook color picker elements, so it really isn't critical.

Reply:
we can have a beer about it some day.

i have had a looot of wasted time on css like this, and it never proved to be worthy to deeply nest things. moreover, nesting creates overly specific selectors which are not always desired and could be a problem later on

Reply:
🍻